packages/components/addon/components/hds/tabs/panel.js

```diff
@@ -0,0 +1,48 @@
+import Component from '@glimmer/component';
+import { cached } from '@glimmer/tracking';
+import { guidFor } from '@ember/object/internals';
+import { action } from '@ember/object';
+
+export default class HdsTabsIndexComponent extends Component {
+  /**
+   * Generates a unique ID for the Panel
+   *
+   * @param panelId
+   */
+  panelId = 'panel-' + guidFor(this);
```

Comment:
[suggestion] let's make a note to switch to `{{unique-id}}` when we're able (https://github.com/emberjs/ember.js/blob/089a021b1b5c5f8ea1cb574fcd841a73af7b2031/packages/%40ember/-internals/glimmer/lib/helpers/unique-id.ts#L6)

Reply:
I'll add a comment, thanks